src/main/java/com/gmailatgauru/dogra/exceltoexcelparsingwebapp/ReadXlsx.java

```diff
@@ -0,0 +1,51 @@
+package com.gmailatgauru.dogra.exceltoexcelparsingwebapp;
+
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class ReadXlsx {
+
+    public void parse(String filePath) throws IOException {
+        FileInputStream file = readFile(filePath);
```

Comment:
Naming. Usually when you see "readFile", you'd expect to get the content of the file, rather than just an input stream

Reply:
I'll change the name